src/dm/dm_bss.cpp

```diff
@@ -234,7 +234,7 @@ void dm_bss_t::encode(cJSON *obj, bool summary)
 
 void dm_bss_t::operator = (const dm_bss_t& obj)
 {
-	strncpy(this->m_bss_info.id.net_id, obj.m_bss_info.id.net_id, strlen(obj.m_bss_info.id.net_id));
+	memmove(this->m_bss_info.id.net_id, obj.m_bss_info.id.net_id, strlen(obj.m_bss_info.id.net_id) + 1);
```

Comment:
While `memmove` does copy a buffer from source to destination, it is not necessarily intended for this use and may have unintended side-effects. Wouldn't `memcpy` be more appropriate? Also since `obj.m_bss_info.id.net_id` is of type `em_long_string_t`, a `sizeof(em_long_string_t)` (similar to the `mac_address_t` memcpys) may be more appropriate.
```suggestion
	memcpy(this->m_bss_info.id.net_id, obj.m_bss_info.id.net_id, sizeof(em_long_string_t));
```